src/task_queue/nudges/user_event_nudge.py

```diff
@@ -43,7 +43,7 @@
 DEFAULT_EVENT_SETTINGS = {
     "when_first_posted": False,
     "within_30_days": True,
-    "within_7_days": True,
```

Comment:
How is within 1 week different from within 7 days.
Also, didn't we plan on changing the wording, `within`, which is confusing to `before` or `after` which are better?

Reply:
1. The DB expects `within_1_week` hence the change.
2. Renaming the text `within` in the options above will require a db migration which is not ideal now. Also, the refinement of this feature(v2) will fix this issue of unclear in the name.